src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -269,77 +318,143 @@ const ListContainer = (props: {
     }
   }, [params?.options?.events, params?.search, params?.type])
 
+  // const onSubmitEditing = (text: string) => text
+
   return (
     <>
-      <TopOfApp
+      {/* <TopOfApp
         showSearchButton
         onSearchButtonPress={() => navigate(screens.search[params.type], '')}
-      />
-      <VStack
-        alignItems="center"
-        maxHeight={windowHeight - heightOfTopOfAppPlusBottomNav}
-        space={4}
-        padding={4}
-      >
-        <Heading size="sm">Projects List</Heading>
-        {/* TODO: reinstate when functionality is ready */}
-        {/* <SegmentedPicker options={projectListOptions} /> */}
-
-        {params?.type && listItemsToShow ? (
-          <>
-            {/* Past / Upcoming / My Events choice */}
-            {params.type === ListType.Events && (
-              <EventOptions selected={eventsSelectedOption} />
-            )}
-
-            {/* Quick search for upcoming events (Today / This week / This month) */}
-            {params.type === ListType.Events &&
-              eventsShowUpcomingQuickSearch &&
-              eventsQuickSearchUpcomingChoice && (
-                <SearchResultsView>
-                  <EventSearchUpcomingQuickSearch
-                    selectedButton={eventsQuickSearchUpcomingChoice}
+      /> */}
+      {/* TODO: reinstate when functionality is ready */}
+      {/* <SegmentedPicker options={projectListOptions} /> */}
+
+      <VStack space={2} alignItems="stretch">
+        <VStack mt="4" px="4">
+          <Heading size="xl" marginBottom="15px">
+            Roles
+          </Heading>
+          <HStack space="2" alignItems="center" width="100%">
+            <View flex={1}>
+              <FreeSearchBar
+                handleSubmit={handleRoleSearchChange}
+                handleChangeText={handleRoleSearchChange}
+                handleClearSearch={handleClearSearch}
+                marginBottom="2"
+              />
+            </View>
+            <View
+              marginBottom="2"
+              height="48px"
+              width="48px"
+              bg="#F0F0F0"
+              borderRadius="10px"
+              padding="2"
+              mx="10px"
+              justifyContent="center"
+              alignItems="center"
+            >
+              <IconButton
+                icon={
+                  <Icon
+                    as={MaterialIcons}
+                    name="tune"
+                    size="24px"
+                    color="gray.500"
                   />
-                </SearchResultsView>
-              )}
+                }
+              />
+            </View>
+          </HStack>
+
+          {/* Display filtered roles based on the search */}
+          <SearchResultsView>
```

Comment:
I am not exactly sure how the search result should be implemented, but when it comes to results I suppose it should be the same style like the other lists and not in this section here. Also searching after every keychange is performancewise a bit costly (and makes the system also slow) so I'd advise here to use something like a debouncing pattern.